src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -494,6 +526,43 @@ private static void ReportDiagnostics(IOperation operation, SmallDictionary<stri
                 _ => null,
             };
 
+        private static IEnumerable<ISymbol> GetPropertyAccessors(IPropertySymbol property, IOperation operation)
```

Comment:
```suggestion
        private static IEnumerable<ISymbol> GetPropertyAccessors(IPropertySymbol property, IPropertyOperation operation)
```

Reply:
From the caller site it is coming as IOperation and used as is, this change will need unnecessary casting